### PR TITLE
CON-3027 Tests(DPxAI): Add test for Quest01 declare-everything

### PR DESCRIPTION
**Pull request by lotoussa**
[CON-3027](https://01talent.atlassian.net/browse/CON-3027)

[CON-3027]: https://01talent.atlassian.net/browse/CON-3027?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: js/tests/declare-everything_test.json
Comment (nprimo) [inline js/tests/declare-everything_test.json:1]: The file name needs to be `declare-everythin.json` to be run by the test image entry point

**Review by nprimo — CHANGES_REQUESTED** [review]
Any special reason why the tests are as `json` file instead of `js`?

**Review by nprimo — APPROVED** [review]
LGTM - to be merged only when the following are approved:
- #2664
- 01-edu/solutions#47

**Review by Oumaimafisaoui — APPROVED** [review]
LGTM

**Pull request merged**